doc/api/punycode.md

```diff
@@ -27,6 +33,9 @@ punycode.encode('☃-⌘'); // '--dqo34k'
 ```
 
 ## punycode.toASCII(domain)
+<!-- YAML
+added: v0.6.2
```

Comment:
@bnoordhuis You are right. I'll fix it right away. I was misled by the punycode doc.

```md
[Punycode.js][] is bundled with Node.js v0.6.2+.
```
I'll fix it, too.